apps/browser/qml/pages/BrowserPage.qml

```diff
@@ -294,7 +294,11 @@ Page {
         onActiveChanged: {
             var isFullScreen = webView.contentItem && webView.contentItem.fullscreen
             if (!isFullScreen && active && !overlay.enteringNewTabUrl) {
-                overlay.animator.showChrome()
+                if (webView.tabModel.count !== 0 || (WebUtils.homePage !== "about:blank" && WebUtils.homePage.length > 0)) {
```

Comment:
@llewelld 
> This looks and works nicely generally. I have a few minor style comments.
> 
> One issue in relation to behaviour is that, on opening the browser when there are no existing tabs, the start page opens full screen. Shortly after the homepage loads and the start page disappears. Is this something that's been fixed elsewhere already? It looks like it's related to [this code](https://github.com/sailfishos/sailfish-browser/blob/master/apps/core/declarativewebcontainer.cpp#L902).

This change resolves the issue when using the home page.

Reply:
Great, this is much better. Thank you.